src/AltinnCore/Designer/Startup.cs

```diff
@@ -272,19 +197,27 @@ public void Configure(IApplicationBuilder appBuilder, IWebHostEnvironment env)
             });
         }
 
-        /// <summary>
```

Comment:
old line 255, new line 180: should ApplicationMetadata be removed from the list of controllers as you have used attribute routing for the endpoints ?

Reply:
This also goes for the routing for DeployController which has been deleted. See line 170

Reply:
Wait for this one please, I have an issue where a couple of requests to certain endpoints can't find their respectful routes. I am still unsure of what is going on, so I'm working on it, I'll update this when I found the problem

Reply:
I've decided to bring back the old attribute routes and remove [controller] from ApplicationMetadata controller, in future PRs we can remove that.